src/arraymancer/laser/tensor/initialization.nim

```diff
@@ -13,7 +13,13 @@ import
   ../private/nested_containers,
   ./datatypes
 # Standard library
-import std / [typetraits, sequtils]
+import std / [typetraits, sequtils, sets]
+
+# The folling export is needed to avoid an compilation error in
+# algorithms.nim/intersection() when running the test_algorithms test:
+# `Error: type mismatch - Expression: items(s1)`
+export sets
```

Comment:
Note that we could also get away with adding a `bind sets.items` inside of `intersection` and `setDiff`. However, given that we now depend on `sets`, I don't see an issue exporting it too.